man/sd_bus_error.xml

```diff
@@ -128,6 +130,17 @@
         <paramdef>const sd_bus_error *<parameter>e</parameter></paramdef>
         <paramdef>const char *<parameter>name</parameter></paramdef>
       </funcprototype>
+
+      <funcprototype>
+        <funcdef>int <function>_sd_bus_error_has_names</function></funcdef>
```

Comment:
hmm, i like the concept, but I am unsure about the underscore naming. i.e. an exported symbol that has an underscore prefix is a bit weird. maybe call this sd_bus_error_has_names_sentinel() or so? i don't think we should prefix it with an underscore, given that it isn#t that internal, it has a man page after all.